test/unit/config.test.ts

```diff
@@ -31,50 +32,91 @@ function createExplorer(overrides: Partial<ReturnType<typeof cosmiconfig>> = {})
   };
 }
 
-suite('config', function () {
-  test('loadConfig() returns undefined when no config exists', async function () {
+suite('loadConfig()', function () {
+  test('returns an Result Err when no config exists', async function () {
     const explorer = createExplorer({
-      load: sinon.fake.rejects(new Error()),
+      load: sinon.fake.rejects(new Error('Not found')),
     });
-    const actual = await loadConfig('./not-found.js', explorer);
-    const expected = undefined;
+    const config = await loadConfig('./not-found.js', explorer);
+    if (config.isOk()) {
+      assert.fail('loadConfig() did not returned an expected error');
+    }
+    const actual = config.error;
+    const expected = 'Not found';
     assert.strictEqual(actual, expected);
   });
 
-  test('loadConfig() returns undefined when config is empty', async function () {
+  test('returns an Result Err when loading does not return an error object', async function () {
```

Comment:
```diff
-returns an Result Err when loading does not return an error object
+returns a Result Err when loading does not return an error object
```

Reply:
Fixed in 2da9cd8d2966fe3ccee8f5e7b798e69e49b2f87b